pkg/appmanager/validateResources.go

```diff
@@ -29,13 +29,26 @@ func (appMgr *Manager) checkValidConfigMap(
 ) (bool, []*serviceQueueKey) {
 	// Identify the specific service being referenced, and return it if it's
 	// one we care about.
+	var keyList []*serviceQueueKey
 	cm := obj.(*v1.ConfigMap)
 	namespace := cm.ObjectMeta.Namespace
 	_, ok := appMgr.getNamespaceInformer(namespace)
 	if !ok {
 		// Not watching this namespace
 		return false, nil
 	}
+	//check as3 config map
+	if as3ok := appMgr.checkAs3ConfigMap(obj); as3ok {
```

Comment:
```suggestion
	if ok = appMgr.checkAs3ConfigMap(obj); ok {
```
We can reuse the same `ok` variable.

Reply:
ok. Taken care